src/main/java/com/bina/varsim/util/Position1.java

```diff
@@ -0,0 +1,20 @@
+package com.bina.varsim.util;
```

Comment:
which should be fine. as long as there is a method to get the the indexing scheme. these classes are pretty much the same except for the 0 vs 1 starting position.